lib/tldr-lint-cli.js

```diff
@@ -89,7 +89,8 @@ cli.process = function(file, args) {
 };
 
 if (require.main === module) {
-  const program = require('commander');
+  const { Command } = require('commander');
+  const program = new Command();
```

Comment:
My only query, is there a reason for not using:
```suggestion
  const { program } = require('commander');
```

I'm happy either way, but just thought I'd check, as we don't have a subcommand.

Reply:
I changed it because the previous version caused an error for me. Maybe some kind of version mismatch.

Reply:
It does need to be changed (as 12.x dropped support for the global export). Just I think the format I used from the docs is a bit cleaner, and makes more sense for the linter (due to it only being a single-command program).

Reply:
Thanks for your feedback. I am really not into JS, so I don't know why I got an error there? Did I use a too old version of node (v20.11.1)? Or of Commando? I think the Commando version should be fixed by the package-lock file, right? Could this change lead to problems in the future?

Reply:
Happy for this to be merged. 👍🏻 Node 20 is fine, and Commando is correctly locked to 12.x 🙂

Reply:
> Happy for this to be merged. 👍🏻 Node 20 is fine, and Commando is correctly locked to 12.x 🙂

Great, it's been a while since the last release (and there have been a few changes and a lot of dependency updates). Can I create a new release for tldr-lint?

Reply:
I think a release is in order. 🥳 I've ran it locally against the `tldr` repo, and all seems to pass. 👍🏻

Reply:
> I think a release is in order. 🥳 I've ran it locally against the `tldr` repo, and all seems to pass. 👍🏻

Done